.git_archival.txt

```diff
@@ -1 +1,4 @@
+node: $Format:%H$
+node-date: $Format:%cI$
+describe-name: $Format:%(describe:tags=true)$
 ref-names: $Format:%D$
```

Comment:
Do we need this line?  the refs name includes both branches and tags and is the source of a lot of trouble with non-reproducible downloads from github...

Reply:
No idea. I just copied from https://github.com/Changaco/setuptools_scm_git_archive where they have that line in the migration guide.

Reply:
Lets defer this question to a later time.  It is a can of worms I do not want to block this moving forward.